src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -1292,4 +1294,1240 @@ void EIRPlantLoopHeatPump::oneTimeInit(EnergyPlusData &state)
+            // Validate fuel type input
+            static constexpr std::string_view RoutineName("processInputForEIRPLHP: ");
+            bool FuelTypeError = false;
+            UtilityRoutines::ValidateFuelTypeWithAssignResourceTypeNum(tempRsrStr, tempRsrStr, thisPLHP.fuelType, FuelTypeError);
+            if (FuelTypeError) {
+                ShowSevereError(state, format("{}{}=\"{}\",", RoutineName, cCurrentModuleObject, thisPLHP.name));
+                ShowContinueError(state, format("Invalid Fuel Type = {}", tempRsrStr));
+                ShowContinueError(state, "Reset the Fuel Type to \"NaturalGas\".");
+                thisPLHP.fuelType = DataGlobalConstants::ResourceType::Natural_Gas;
+                errorsFound = true;
+                FuelTypeError = false;
+            }
+
+            // A7 End use category
+            thisPLHP.endUseSubcat = UtilityRoutines::MakeUPPERCase(fields.at("end_use_subcategory").get<std::string>());
+            if (thisPLHP.endUseSubcat == "") {
+                thisPLHP.endUseSubcat = "Heat Pump Fuel Fired"; // or "General"?
+            }
+
+            // N1 Nominal heating capacity
+            auto &tmpRefCapacity = fields.at(refCapFieldTag);
+
+            if (tmpRefCapacity == "Autosize") {
+                thisPLHP.referenceCapacity = DataSizing::AutoSize;
+                thisPLHP.referenceCapacityWasAutoSized = true;
+            } else {
+                thisPLHP.referenceCapacity = tmpRefCapacity.get<Real64>();
+            }
+
+            // N2 Nominal heating capacity
+            thisPLHP.referenceCOP = fields.at("nominal_cop").get<Real64>();
+            if (thisPLHP.referenceCOP <= 0.0) thisPLHP.referenceCOP = 1.0;
+
+            // N3 Design flow rate
+            auto &tmpFlowRate = fields.at("design_flow_rate");
+            if (tmpFlowRate == "Autosize") {
+                thisPLHP.loadSideDesignVolFlowRate = DataSizing::AutoSize;
+                thisPLHP.loadSideDesignVolFlowRateWasAutoSized = true;
+            } else {
+                thisPLHP.loadSideDesignVolFlowRate = tmpFlowRate.get<Real64>();
+            }
+
+            // GAHP: Add a default source side flow rate, not from input
+            Real64 defDummyASDesVolFlowRate = 1.0;
+            thisPLHP.sourceSideDesignVolFlowRate = defDummyASDesVolFlowRate;
+
+            // N4 Design supply temperature
+            auto &tmpDesSupTemp = fields.at("design_supply_temperature");
+            if (tmpDesSupTemp == "Autosize") {
+                // sizing
+            } else {
+                thisPLHP.desSupplyTemp = tmpDesSupTemp.get<Real64>();
+            }
+
+            // N5 Design temperature lift
+            auto &tmpDesTempLift = fields.at("design_temperature_lift");
+            if (tmpDesTempLift == "Autosize") {
+                // sizing
+            } else {
+                thisPLHP.desTempLift = tmpDesTempLift.get<Real64>();
+            }
```

Comment:
Maybe this is expected, but this is never used anywhere, well it has a default of `11.1` and is `autosizable` per the IDD. @jcyuan2020

Reply:
This is an input field that is currently not in use (or not finding an elegant way about how to use it). It seems to be a a field that might be usable in sizing--but could get some complications with low and other targeting setpoint of the loop at the first glance.